cheetahcrawl.css

```
@@ -0,0 +1,305 @@
+.cc-panel{
+  display:none;
+  flex-direction:column;
+  align-items:center;
+  width:100%;
+  max-width:720px;
+}
+.cc-panel.show{display:flex;}
+
+/* ---------- race ---------- */
+.cc-countdown{
+  font-family:'Chillax',sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Chillax" (font-family-name-quotes)

(font-family-name-quotes)